ui/src/adapters/api/credentials.ts

```diff
@@ -86,17 +91,19 @@ export async function getCredential({
 
 export async function getCredentials({
   env,
-  params: { did, query, status },
+  params: { did, max_results, page, query, status },
```

Comment:
I'd avoid having snake case params in our interfaces. If we need to send it to the API I'd name this param `maxResults` and later send the `max_results` name to the API.

Reply:
Yep, changed!